build.gradle.kts

```diff
@@ -29,10 +28,8 @@ dependencies {
 
     implementation(kotlin("stdlib-jdk8", kotlinVersion))
     implementation(kotlin("reflect", kotlinVersion))
-    implementation("com.google.guava:guava:27.1-jre")
-    implementation("com.pinterest:ktlint:0.32.0")
-    // implementation("com.google.flogger:flogger:$floggerVersion")  -- Can' t use until log4j2 is supported
-    // implementation("com.google.flogger:flogger-system-backend:$floggerVersion")  -- Can 't use until log4j2 is supported
+    implementation("com.google.guava:guava:28.2-jre")
+    implementation("com.pinterest:ktlint:0.36.0")
```

Comment:
This feels more like a build script dep to me. What’s it for?

Reply:
Sounds like you're saying this should be a dev dependency, instead of a compile dependency. I agree. It's just a Kotlin linter that ensures you're using idiomatic Kotlin. I do need to double-check that this is configured correctly though

Reply:
Yep yep. I believe gradle lets you have them at the top in `buildScript` chunk?

Reply:
Still not used to the gradle terminology. Looking up about buildscript dependencies. But I think you're right. Will fix

Reply:
According to the documentation, it says it's supposed to be in the dependencies section.
https://github.com/pinterest/ktlint